k2/csrc/algorithms.h

```diff
@@ -121,17 +121,40 @@ class Renumbering {
     return new2old_;
   }
 
+  /* Return a mapping from new index to old index, with one extra element
+     containing the total number of kept elements if extra_element == true.
+     If Keep() can be interpreted as a tails vector, i.e. with 1 at the end
+     of sub-lists of elements, then New2Old(true) would corresponds to a
+     row-splits array and Old2New(false) would correspond to a row-ids
+     array.
+  */
+  Array1<int32_t> New2Old(bool extra_element) {
```

Comment:
Should we make the interface the same as `Old2New(bool extra_element = false)`?
Just for consistency.

Reply:
It is the same in effect; there is an overloaded version taking no arg, that doesn't return the extra element, but
returns the ref.  If we had default arg, it would cause ambiguity in function overloading.

Reply:
I mean, removing the one with no argument.

Reply:
RE removing the one with no argument: the reason it's designed the way it is, is to avoid redundant copies of tensors, which could take time at runtime and also bulk up the compiled code with unnecessary instructions.  New2Old() is called quite a lot so I thought it was worth saving that time.